site/content/_index.md

```diff
@@ -1,6 +1,4 @@
 ---
-title: "Welcome to the NGINX Agent documentation"
+title: "NGINX Agent Documentation"
```

Comment:
is the title case intentional?

Reply:
Yes. It's the landing page for the entire documentation website, and I'm treating the documentation itself as a product in regard to capitalising a proper noun.